recipes/vscode-jupyter/build.sh

```diff
@@ -0,0 +1,5 @@
+npm install -g vsce
+npm ci
+npm run clean
+npm run package
+code-server --install-extension ms-toolsai-jupyter-insiders.vsix
```

Comment:
Probably the `--extension-dir` still has to be adjusted to `$PREFIX/share/code-server/extensions`